frontend/src/components/common/form/SearchToggle.tsx

```diff
@@ -0,0 +1,83 @@
+import { ReactComponent as MapSvg } from 'assets/images/icon-map.svg';
+import { ReactComponent as TableSvg } from 'assets/images/icon-table.svg';
+import React from 'react';
+import { useHistory } from 'react-router';
+import styled from 'styled-components';
+
+import TooltipWrapper from '../TooltipWrapper';
+import { ButtonProps } from '.';
+
+export enum SearchToggleOption {
+  /** The map is the active page */
+  Map = 'map',
+  /** Property List View is the active page */
+  List = 'list',
+}
+
+interface ISearchToggleProps extends ButtonProps {
+  /** set the id of the tool tip use for on hover of the plus buttons */
+  toolId: string;
+  /** Which toggle view is currently active */
+  toggle?: SearchToggleOption;
+}
+
+/**
+ * SearchToggle displaying two buttons which act like a toggle to navigate from the map to the property list view (and vise-versa).
+ * @param param0
+ */
+export const SearchToggle: React.FC<ISearchToggleProps> = ({
+  toggle = SearchToggleOption.Map,
+  toolId,
+  ...props
+}) => {
+  const history = useHistory();
+  const handleClick = () => {
+    if (toggle === SearchToggleOption.Map) history.push('/properties/list');
+    else history.push('/mapview');
+  };
+
+  return (
+    <StyledToggle>
```

Comment:
Not a big fan of the design, as one button will always be disabled.  It should just be a single button that shows one or the other... but I'm just doing what is in the story.

Reply:
better to bring up these issues in the design meeting.

Reply:
I get the frustration of communicating during some meetings but presumably that doesn't extend to Bobbi.

Reply:
of course, these things get missed in the design meeting sometimes too, Bobbi is pretty good about ad-hoc design meetings during sprint as well.

Reply:
The comment was to explain why there are two buttons that are acting as a toggle, nothing more.  Sprint is done today, so having a design change that has no impact to functionality isn't something I would recommend.  Instead I've provided feedback in the PR that may spur further conversation between the dev team and the UI/UX.  Up to you guys if you want to chase it down in the future... But for now I'm just doing what's in the story.